features/models.py

```diff
@@ -147,3 +147,34 @@ class Meta:
                 fields=["feature", "tag"], name="unique_feature_tag"
             ),
         ]
+
+
+class ContactInfo(models.Model):
+
```

Comment:
This blank line doesn't seem to match style elsewhere in this project. Not sure if I can sleep properly after seeing this.

Reply:
well, now I won't be able to sleep either 😢

Reply:
barely slept 😪

Reply:
Sorry for this, I'll fix it right away!